### PR TITLE
Fixing Example to be functional in ReadMe

### PR DESCRIPTION
**Pull request by cdesch**
Fixing the example to be copy-and-paste functional. (nice when deploying through nuget and trying out the library for the first time)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,14 +30,14 @@ Barcode (string, BarcodeLib.TYPE);
...
+BarcodeLib.Barcode b = new BarcodeLib.Barcode(BarcodeLib.TYPE.UPCA, 
           "038000356216", Color.Black, Color.White, 300, 150);
 ```
 ### Example ###
 ```
-BarCodeLib.Barcode b = new BarCodeLib.Barcode(BarCodeLib.TYPE.UPCA, 
-          "038000356216", Color.Black, Color.White, 300, 150);
-Image img = b.Encode();
+BarcodeLib.Barcode b = new BarcodeLib.Barcode();
+b.Alignment = BarcodeLib.AlignmentPositions.CENTER;
```
Comment (barnhill) [inline README.md]: for the example you can remove this line as its not needed for a simple example.

Wow I havent updated that readme in quite some time it appears.  Good job noticing and thanks for the pull.  Correct that one small thing and Ill merge it.

**Comment by cdesch** [comment]
I removed the line.

**Comment by cdesch** [comment]
Thanks @barnhill !!

Love the library. I have a code sample for adding the barcode to a receipt (printer doc) if you want me to make a pull request with it. That is kind of going on the assumption that a lot of people are generating barcodes for basic printing. C# isn't my primary language, so it might not be perfect.

**Comment by barnhill** [comment]
Want me to take a look at your code in GitHub before you go to the trouble
to pull?

**Comment by cdesch** [comment]
Sure. I think it might be better off as it's own wiki page if you choose to use a whole class. I need to do some cleaning up of it in my own project to make it more "DRY". There is also a QRCode in there as well. I'm on the fence on whether I should make it into more of a drop in class sample or a smaller procedural sample. 

Class Style: https://gist.github.com/cdesch/aebbf85a5360131e74691e9043565d10 
Smaller Sample: https://gist.github.com/cdesch/049cb4f924f0d613379744581ce6ab7e

**Comment by barnhill** [comment]
Im very open to including a folder called Samples and put whole projects in
there that use the library for examples for people to get started.  I also
need some documentation on the library but I just dont have the time to
write it up.

**Pull request merged**